services/cii-best-practices/cii-best-practices.service.js

```diff
@@ -0,0 +1,162 @@
+'use strict'
+
+const Joi = require('joi')
+const BaseJsonService = require('../base-json')
+const {
+  colorScale,
+  coveragePercentage: coveragePercentageColor,
+} = require('../../lib/color-formatters')
+
+const ciiBestPracticesSchema = Joi.object({
+  badge_level: Joi.string().required(),
+  tiered_percentage: Joi.number().required(),
+}).required()
+
+const keywords = ['cii', 'cii best practices', 'core infrastructure initiative']
+
+module.exports = class CIIBestPracticesService extends BaseJsonService {
+  static render({ label, message, color }) {
+    return {
+      label,
+      message,
+      color,
+    }
+  }
+
+  static get defaultBadgeData() {
+    return { color: 'blue', label: 'cii' }
+  }
+
+  static get category() {
+    return 'other'
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'CII Best Practices Level',
+        namedParams: {
+          metric: 'level',
+          projectId: '1',
+        },
+        pattern: 'level/:projectId',
+        staticExample: this.render({
+          label: 'cii',
```

Comment:
I don't think you have to repeat `cii` here, nor have the `label` parameter in the render function. This is specified as part of the default badge data.